src/emqx_broker.erl

```diff
@@ -232,6 +232,9 @@ forward(Node, To, Delivery) ->
         {badrpc, Reason} ->
             ?ERROR("[Broker] Failed to forward msg to ~s: ~p", [Node, Reason]),
             Delivery;
+        {badtcp, Reason} ->
+            ?ERROR("[Broker] Failed to forward msg to ~s: ~p", [Node, Reason]),
+            Delivery;
         Delivery1 -> Delivery1
```

Comment:
{Error, Reason} ->
    ?ERROR("[Broker] Failed to forward msg to ~s: ~p ~p", [Node,Error, Reason]),
            Delivery;

Reply:
This will capture a normal return value of the RPC evaluation result (if the result is a two element tuple.